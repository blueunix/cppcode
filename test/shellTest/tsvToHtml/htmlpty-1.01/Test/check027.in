<ul><li>
<b>When is a Functional Program Not a Functional Program?</b><br>
John Longley (University of Edinburgh, Scotland)
</li>
</ul>
<ol><li>
<b>When is a Functional Program Not a Functional Program?</b><br>
John Longley (University of Edinburgh, Scotland)
</li>
</ol>
<dl>
<dt>dt-stuff
<dd>dd-stuff
<dt>more-dt-stuff
<dd>more-dd-stuff
</dl>
    <dd>
    <nobr><a HREF="papers/FengPeng00/">
    Accelerating Accurate B-spline Free-form Deformation of Polygonal Objects.</a>
    Jieqing Feng and Qunsheng Peng.
    </nobr></dd>
