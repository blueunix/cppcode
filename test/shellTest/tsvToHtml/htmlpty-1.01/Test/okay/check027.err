:22:non-zero <TAG> ... </TAG> nesting level at end-of-file
