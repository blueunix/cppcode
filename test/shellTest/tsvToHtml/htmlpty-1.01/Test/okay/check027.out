<UL>
    <LI>
        <B>When is a Functional Program Not a Functional Program?</B>
        <BR>
        John Longley (University of Edinburgh, Scotland)
    </LI>
</UL>
<OL>
    <LI>
        <B>When is a Functional Program Not a Functional Program?</B>
        <BR>
        John Longley (University of Edinburgh, Scotland)
    </LI>
</OL>
<DL>
    <DT>
        dt-stuff
    </DT>
    <DD>
        dd-stuff
    </DD>
    <DT>
        more-dt-stuff
    </DT>
    <DD>
        more-dd-stuff
    </DD>
</DL>
<DD>
<NOBR><a HREF="papers/FengPeng00/">
    Accelerating Accurate B-spline Free-form Deformation of Polygonal Objects.</a>
    Jieqing Feng and Qunsheng Peng.
</NOBR>
</DD>
